man/man8/zfs-send.8

```diff
@@ -40,6 +40,7 @@
 .Nm zfs
 .Cm send
 .Op Fl DLPRbcehnpsvw
+.Op Fl X Ar dataset[,dataset,...]
```

Comment:
```
.Op Fl X Ar dataset Ns Oo , Ns Ar dataset Oc Ns …
```

Reply:
I think `.Op Fl X Ar dataset Ns Oo , Ns Ar dataset Ns ... Oc` is better -- that produces the ellipses inside the inner brackets.

Reply:
Which is (a) inconsistent (cf. zfs-userspace.8) and (b) wrong – you repeat `, Ar dataset`, not `Ar dataset`

Reply:
Alrighty.  Done